src/data/accessibility-quiz.ts

```diff
@@ -1,5 +1,14 @@
 const accessibilityQuiz = [
   {
+    Question: "According to W3C,which is not the valid guideline for creating style sheets that promote accessibility?",
```

Comment:
I think this would be better : "According to the W3C, which of the following is not a valid guideline for crafting style sheets that enhance accessibility?"
Ans: "Utilize the 'px' unit to define font sizes."
 1: "Ensure there is always a fallback generic font."
 2: "Opt for numbers instead of names when specifying colors."
 3: "Ensure consistent 'class' names for the same concept across all style sheets."
Exp: "For setting font sizes, use the 'em' unit and favor relative length units and percentages."

Reply:
@Gautam-Hegde 

Its a good suggestion 👍 

Also, for future reviews, you can also use the suggestion tool to apply your changes
It makes it easier for people to commit those suggested changes 👍 

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request

Reply:
i see, i will then modify the question as per your suggestion and send back the pr